docs/source/hsm.md

```diff
@@ -89,31 +108,31 @@ in the `signcerts` folder to retrieve the private key from inside the HSM.
 The process for creating the MSP folders will be different depending on if you
 are using a Fabric Certificate Authority (CA) your own CA.
 
-### Using a Fabric CA
+### Using an HSM with a Fabric CA
+
+You can set up a Fabric CA to use an HSM by making the same edits to the CA server configuration file as you would make to a peer or ordering node. Because you can use the Fabric CA to generate keys inside an HSM, the process of creating the local MSP folders is straightforward. Use the following steps:
+
+1. Create an HSM token and point to it in the Fabric CA server configuration file. When the Fabric CA server starts, the HSM described in the configuration file will generate the CA signing certificate and store the private key inside the HSM. If you are not concerned about exposing your CA signing certificate, you can skip this step and only configure an HSM for your peer or ordering nodes.
 
-You can set up a Fabric CA to use an HSM by making the same edits to the
-configuration file as you would make to a peer or ordering node. Because you can
-use Fabric CA to generate keys inside an HSM, the process of creating the local
-MSP folders is straightforward. Use the following steps:
+2. Use the Fabric CA client to register the peer or ordering node identities with your CA.
 
-1. Create an HSM token and point to it in the Fabric CA server configuration
-file. When the Fabric CA server starts, it will generate the CA signing
-certificate inside your HSM. If you are not concerned about exposing your CA
-signing certificate, you can skip this step.
+3. Before you deploy a peer or ordering node with HSM, you need to enroll the node admin, storing its private key in the HSM. Edit the BCCSP section of the Fabric CA client config file or associated environment variables to point to the HSM configuration for your peer or ordering node. In the Fabric CA Client configuration file, replace the default `SW` configuration with the `PKCS11` configuration and providing the values for your own HSM:
```

Comment:
`Before you deploy a peer or ordering node with HSM support, you need to enroll the node admin by storing its private key in the HSM.`

`configuration and provide the values for your own HSM:`